src/utils/atoms/createAsyncAtom/createAsyncAtom.ts

```diff
@@ -165,6 +169,8 @@ export function createAsyncAtom<
               log('5.1.B.1. Error');
               dispatch(create('_error', params, ABORT_ERROR_MESSAGE));
             } else if (ctx.abortController === abortController) {
+              ctx.abortController = null;
```

Comment:
`delete ctx.abortController;` will do the same as `ctx.abortController = null;`?

Reply:
```suggestion
              delete ctx.abortController;
```

Reply:
yep

Reply:
let's do it in one approach. 
If we go with devete, could you also please update in line 201?